week3/todoCLI.js

```diff
@@ -0,0 +1,115 @@
+const readline = require('readline');
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+let myStuff = []; 
+//The above "myStuff" array will be storing the todo's that a user enters.
+function userInput(answer){
+    if (answer === 'v'){
+        viewList();
+    } else if (answer === 'n'){
+        addNewItem();
+    } else if (answer[0] === 'c') {
+        completeItem(answer);
+    } else if (answer[0] === 'd') {
+        deleteItem(answer);
+    } else if (answer === 'q'){
+        quitList();
+    } else {
+        console.log(`You must enter one of the following: v, n, cX (where X is a number), dX (where X is a number), q`);
+        menu();
+    };
+};
+
+function menu () {rl.question(
+    `Welcome to ToDo CLI!\n--------------------\n(v) View ∙ (n) New ∙ (cX) Complete ∙ (dX) Delete ∙ (q) Quit\n>`, 
+    userInput)};
+    menu();
+
+function viewList (){
+    console.log(`You're viewing your to-do list`);
+    if (myStuff.length === 0) {
+         console.log('but your to-do list is empty')
+    } else {
+    for (let i = 0; i < myStuff.length; i++){
+            if (myStuff[i][0] === false) {
+                console.log (`${i}  [ ] ${myStuff[i][1]}`)
+                } else if (myStuff[i][0] === true) {
+                    console.log(`${i}  [✓] ${myStuff[i][1]}`)
+           }
+        }
+    };
+    menu();
+};
+
+function addNewItem (){
+    console.log(`you're adding a new to-do item`);
+    rl.question(`What task would you like to add?\n`, answer =>{
+        myStuff.push([false, answer]);
+        menu();
+    });
+};
+
+function completeItem (answer) {
+    let userInputNumber = '';
+    for (let i = 1; i<answer.length; i++){
+        userInputNumber += answer[i]
+    };
```

Comment:
This for loop essentially slices off the first character within the answer string.
Can also use 
```js
const userInputNumber = answer.slice(1);
```